lib/commands/init.js

```diff
@@ -107,17 +107,17 @@ module.exports = Command.extend({
         }
       })
       .then(function() {
-        if (!commandOptions.skipNpm) {
-          return npmInstall.run({
-            verbose: commandOptions.verbose,
-            optional: false
+        if (!commandOptions.skipBower) {
+          return bowerInstall.run({
+            verbose: commandOptions.verbose
```

Comment:
oh okay, I missed that bower and npm switched places. I just saw that the `optional` option was removed, but it looks like it just moved further down.